Multisig/UI/Safe Management/Create Safe/GoogleWeb3AuthLoginModel.swift

```diff
@@ -0,0 +1,38 @@
+import Foundation
+import CustomAuth
+import UIKit
+
+class GoogleWeb3AuthLoginModel {
```

Comment:
I would generalized the name to handle all Web2Auth types, Apple and Google, we have

Reply:
They might be very different. I will unify, when Apple is sufficiently similar, ok?